kurtschlueter/lockerta
# Latest news

Ended up getting a lot on this commit:
- News image from slideshow along with header for latest news
- Basic latest news container setup for responsiveness
- Got working social media links in the footer after finding the appropriate pages
- Get amazon s3 and paperclip gems up and running
- Set up amazon s3 bucket (this will need to be done again with Nate's actual info so he can control it, currently on my CC - make sure not to push keys)
- Set up dotenv gem so we can stash keys and secret stuff outside of the actual code
- Get images from actual articles
- Change article model so it included the appropriate s3/paperclip attachment files 
- Add aws and paperclip initializers so push/pull from the appropriate s3 regions and links
- Format the articles section of the latest news section according to screenshots from Nate
- Move repetitive ratings/sections of the home page into partials to clean stuff up
- Add subtitles to the articles model/database since they are in the view and weren't available
